app/Controllers/PasswordController.php

```
@@ -59,32 +59,52 @@ public function forgot(Request $request, Response $response, mysqli $db): Respon
+                    '<p>' . __('Oppure copia e incolla questo link nel tuo browser:') . '</p>' .
+                    '<p><code style="background-color: #f3f4f6; padding: 10px; display: block; word-break: break-all;">' . htmlspecialchars($resetUrl, ENT_QUOTES, 'UTF-8') . '</code></p>' .
+                    '<p><strong>' . __('Nota:') . '</strong> ' . __('Questo link scadrà tra 2 ore.') . '</p>' .
+                    '<p>' . __('Se non hai richiesto il reset della password, puoi ignorare questa email. Il tuo account rimane sicuro.') . '</p>';
+
+                // Use EmailService if available, fall back to Mailer
+                try {
+                    $emailService = new EmailService($db);
+                    $emailService->sendEmail($email, $subject, $html, $name);
+                } catch (\Throwable $e) {
+                    error_log('EmailService failed, falling back to Mailer: ' . $e->getMessage());
+                    Mailer::send($email, $subject, $html);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\Mailer' in method 'forgot'. (undefined)

(StaticAccess)